### PR TITLE
*: flag refactors

### PR DESCRIPTION
**Pull request by KaloyanTanev**
- Change Obol's API default URL in flags to include `/v1`
- validator-pubkey to validator-public-key in flag description
- Rename test command measurement from Bad to Poor
- Increase test peer unit tests timeout
- bump ethdo version in compose tests to 1.35.2

category: misc
ticket: none

### CODE REVIEW

File: cmd/exit.go [hunk 1]
```diff
@@ -137,7 +137,7 @@ func bindExitFlags(cmd *cobra.Command, config *exitConfig, flags []exitCLIFlag)
 		case publishTimeout:
 			cmd.Flags().DurationVar(&config.PublishTimeout, publishTimeout.String(), 30*time.Second, "Timeout for publishing a signed exit to the publish-address API.")
 		case validatorIndex:
-			cmd.Flags().Uint64Var(&config.ValidatorIndex, validatorIndex.String(), 0, "Validator index of the validator to exit, the associated public key must be present in the cluster lock manifest. If --validator-pubkey is also provided, validator liveliness won't be checked on the beacon chain.")
+			cmd.Flags().Uint64Var(&config.ValidatorIndex, validatorIndex.String(), 0, "Validator index of the validator to exit, the associated public key must be present in the cluster lock manifest. If --validator-public-key is also provided, validator liveliness won't be checked on the beacon chain.")
```
Comment (OisinKyne) [inline cmd/exit.go:140]: Sorry for the small nit, would make the same tweak on the `validator-public-key` description. 'Liveliness' is a strange word to use. 

Sidenote: If you provide an index and no public key, will it fetch the validator from the chain, get its public key, check for its presence in the lockfile and continue gracefully? I assume yes. 

```suggestion
			cmd.Flags().Uint64Var(&config.ValidatorIndex, validatorIndex.String(), 0, "Validator index of the validator to exit, the associated public key must be present in the cluster lock manifest. If --validator-public-key is also provided, validator existence won't be checked on the beacon chain.")
```

File: cmd/test.go [hunk 2]
```diff
@@ -105,7 +105,7 @@ const (
 	// measurement tests
 	testVerdictGood testVerdict = "Good"
 	testVerdictAvg  testVerdict = "Avg"
```
Comment (OisinKyne) [inline cmd/test.go:107]:
```suggestion
	testVerdictAvg  testVerdict = "Okay"
```

**Review by OisinKyne — APPROVED** [review]
My suggestion about preferring "Okay" instead of "Average" can be ignored until a future PR. Again its a nitpick, we don't really know what the average latency is to a relay, so okay is a more vague word than average. :)

File: cmd/exit.go (hunk 1 above)
Comment (KaloyanTanev) [reply cmd/exit.go]: Yes, that's correct!

File: cmd/test.go (hunk 2 above)
Comment (KaloyanTanev) [reply cmd/test.go:107]: Let's revise those again when we improve the whole scoring system? Then we can have poor, average, okay, perfect, etc.

**Pull request merged**